wiki/Beatmap/Guest_difficulty/en.md

```diff
@@ -11,6 +10,16 @@ tags:
 
 *For regulations surrounding guest difficulties, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username. A beatmap featuring multiple creators is also known as a [collaborative difficulty](/wiki/Beatmap/Beatmap_collaborations).
+A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name which contains the guest creator's username.
```

Comment:
very nitpicky/technical but "containing" isn't the same as "which contains" here and I think the verb was intentional. the former makes "difficulty name containing the guest creator's username" the way to identify GDs, while the latter reduces it to just "difficulty name", followed by explanation of what's in the diff name. the latter also makes a definite statement abt what to expect in the diff name, even though this sentence begins with "can *usually* be identified"

so basically...
```suggestion
A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```